packages/components/src/logos/woo-logo/index.tsx

```diff
@@ -11,6 +11,7 @@ export const WooLogo = ( props: React.SVGProps< SVGSVGElement > ) => (
 		viewBox="0 0 183.6 47.5"
 		{ ...props }
 	>
+		<title>WooCommerce logo</title>
```

Comment:
There's a related effort DS-216 to standardize all of our logos. I think this addition is consistent with that (using `title` for text alternative), but we may want to leave out the word "logo" for [consistency](https://github.com/Automattic/wp-calypso/blob/ceb55d4369630b916e14c0a1acc50b82a51512b2/packages/components/src/logos/jetpack-logo/index.tsx#L79) and to [avoid redundancies](https://webaim.org/techniques/alttext/#logos).

```suggestion
		<title>WooCommerce</title>
```